cajita/src/Cajita_Array.hpp

```diff
@@ -655,6 +659,7 @@ struct Norm1Functor
   \param norms The norms for each degree-of-freedom in the array. This vector
   should be pre-sized to the number of degrees-of-freedom per entity.
 */
+#ifndef KOKKOS_ENABLE_OPENMPTARGET // FIXME_OPENMPTARGET
```

Comment:
In general, why do the vector reductions not work?

Reply:
Missed this one - these are also MD reductions, right?